spec/discoverAndQueuePdrs/DiscoverAndQueuePdrs.input.template.json

```diff
@@ -11,9 +11,12 @@
       "internal": ""
     },
     "collection": {
+      "files": [],
       "provider_path": "cumulus-test-data/pdrs",
       "name": "test-collection",
-      "granuleIdExtraction": "(.*)"
+      "granuleIdExtraction": "(.*)",
+      "granuleId": "(.*).hdf",
```

Comment:
What's the difference between `granuleIdExtraction` and `granuleId`?

Reply:
I think granuleIdExtraction is the pattern used to extract a granule id, and granuleId shows the format of that granule id.  Without them, the IngestGranule workflow will fail.

Reply:
Actually, it's ParsePdr.  After a PDR is discovered, it will queue ParsePdr execution for this PDR, and cumulus message queued will contain the collection/provider information.  Without granuleIdExtraction and granuleId information, the ParsePdr execution will fail.

Reply:
I've searched all through the ParsePdr code, and I can't find anywhere that it's referencing that `granuleId` setting.  Can you please include a link from github?

Reply:
I copied the config from https://github.com/cumulus-nasa/cumulus-integration-tests/blob/master/spec/ingestGranule/IngestGranule.input.template.json
I am going to remove granuleId from the files to see if the tests still work.

Reply:
@yjpa7145 https://github.com/cumulus-nasa/cumulus/blob/master/packages/api/models/collections.js#L37
https://github.com/cumulus-nasa/cumulus/blob/master/packages/api/models/schemas.js#L48
granuleId is in the schema.